### PR TITLE
Remove support for timing in scala.App entry point

### PR DESCRIPTION
**Pull request by fbertra**
FIX https://github.com/scala-native/scala-native/issues/302 "Add support for scala.App"

### CODE REVIEW

File: scalalib/overrides-2.11/scala/App.scala [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+ *  `args` returns the current command line arguments as an array.
+ *
+ *  ==Caveats==
+ *
+ *  '''''It should be noted that this trait is implemented using the [[DelayedInit]]
+ *  functionality, which means that fields of the object will not have been initialized
+ *  before the main method has been executed.'''''
+ *
+ *  It should also be noted that the `main` method should not be overridden:
+ *  the whole class body becomes the â€œmain methodâ€�.
```
Comment (soc) [inline scalalib/overrides-2.11/scala/App.scala]: mojibake!

**Comment by densh** [comment]
Apart from the issue that @soc pointed out, it looks good to me.

**Comment by fbertra** [comment]
@soc, the comments are from the original scalalib file and all credits are due to Martin Odersky.  See https://github.com/scala/scala/blob/2.11.x/src/library/scala/App.scala.

I only removed the lines of code dealing with time.

@densh Do we really want to change these comments?

**Comment by densh** [comment]
There seems to be some issue with text encoding somewhere as the original text is `“main method”` and the modified one is `â€œmain methodâ€�`.

**Comment by fbertra** [comment]
Ok, I see it now.

**Comment by densh** [comment]
@fbertra Can you please rebase on top of master and reformat with `bin/scalafmt --version 0.4.7` ?

**Comment by fbertra** [comment]
I'm on it

**Comment by densh** [comment]
LGTM!

**Comment by fbertra** [comment]
Thanks @soc for the comment and sorry for misunderstanding it at first.

The original scala.App file has an unrecognizable encoding for my editor (jEdit) and opening it with utf-8 encoding creates the strange caracters

And note to self: never again answer github comments from smartphone.

**Pull request merged**